be/src/main/java/yeonba/be/login/controller/LoginController.java

```diff
@@ -69,17 +75,13 @@ public ResponseEntity<CustomResponse<UserIdInquiryResponse>> idInquiry(
         .body(new CustomResponse<>(new UserIdInquiryResponse("mj3242@naver.com")));
   }
 
-  @Operation(
-      summary = "비밀번호 찾기",
-      description = "이메일로 임시 비밀번호를 발급받을 수 있습니다."
-  )
-  @ApiResponse(
-      responseCode = "204",
-      description = "임시 비밀번호 발급(비밀번호 찾기) 정상 처리"
-  )
+  @Operation(summary = "비밀번호 찾기", description = "이메일로 임시 비밀번호를 발급받을 수 있습니다.")
+  @ApiResponse(responseCode = "202", description = "임시 비밀번호 발급(비밀번호 찾기) 정상 처리")
   @PostMapping("/users/help/pw-inquiry")
   public ResponseEntity<CustomResponse<Void>> passwordInquiry(
-      @RequestBody UserPasswordInquiryRequest request) {
+      @RequestBody @Valid UserPasswordInquiryRequest request) {
+
```

Comment:
제가 생각할 때는 `@RequestBody UserPasswordInquiryRequest`를 하나로 보고 이것을 `@Valid`로 검증한다라는 의미로 생각하는게 자연스러운데 어떻게 생각하시나요?
```suggestion
      @Valid @RequestBody UserPasswordInquiryRequest request) {

```

Reply:
로직 흐름상 더 자연스러운 것 같습니다. 반영하겠습니다!